mmv1/products/cloudbuild/api.yaml

```diff
@@ -146,6 +146,13 @@ objects:
               The branch, tag, arbitrary ref, or SHA version of the repo to use when resolving the 
               filename (optional). This field respects the same syntax/resolution as described here: https://git-scm.com/docs/gitrevisions 
               If unspecified, the revision from which the trigger invocation originated is assumed to be the revision from which to read the specified path.
+          - !ruby/object:Api::Type::String
+            name: 'githubEnterpriseConfig'
+            exactly_one_of:
+              - git_file_source.0.github_enterprise_config
```

Comment:
I think we can safely remove this? I assume this is not a required field. (Plus, with `bitbucket_server_config ` implemented before, we probably should've used `conflicts` instead of `exactly_one_of`, as I don't think it's a required union field. Sorry, my mistake)

Reply:
We do plan on adding the bitbucket config soon. We could remove it, but it will be re-added later if so. What do you think is best?

Reply:
Ah, I see what youre saying. That makes sense. I removed it. When adding bitbucket later I'll remember to use `conflicts`.